templates/terraform/examples/cloud_run_domain_mapping_basic.tf.erb

```diff
@@ -10,7 +10,7 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
   template {
     spec {
       containers {
-        image = "gcr.io/cloudrun/hello"
+        image = "us-docker.pkg.dev/cloudrun/container/hello"
```

Comment:
What's the benefit from changing the default image, exactly?

Reply:
So GCR is going away in favour of AR. The cloud run UI now shows this container instead.

Reply:
Got it. Thanks!